src/test.c

```diff
@@ -28,6 +30,45 @@ struct Table* retrieve(struct ParseTree* tree) {
   return NULL;
 }
 
+void testCreateTable(void) {
```

Comment:
This whole function is potentially unclear. Guys let me know if I should comment it further.